docs/multiform.rst

```diff
@@ -60,6 +60,13 @@ use an OrderedDict to define the :attr:`~MultiForm.form_classes`. ::
             ('profile', ProfileForm),
         ))
 
+An implementation of the attribute :attr:`~MultiForm.base_fields` is provided,
+but the keys are a combination of the subform keys and the field names rather
+than just the field names. The reason an implemention is provided at all, given
+it is merely an approximation, is interoperability with ``WizardView``, which
+introspects the values of :attr:`~MultiForm.base_fields`.  See
+:ref:`wizardview_guide`.
+
```

Comment:
In the (albeit quite rare) case that they do encounter this issue, I think it will be an extremely difficult bug to  troubleshoot. I don't think it would hurt to try to be a little helpful here.